merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/PostgresPlanRepository.java

```diff
@@ -326,8 +327,8 @@ private static PlanDatasetRecord associatePlanWithDataset(
   private static <T> T parseJson(final String subject, final JsonParser<T> parser)
   throws InvalidJsonException, InvalidEntityException
   {
-    try {
-      final var requestJson = Json.createReader(new StringReader(subject)).readValue();
+    try (JsonReader reader = Json.createReader(new StringReader(subject))) {
```

Comment:
I think this is actually a no-op, as `JsonReader` simply closes the underlying reader, here a `StringReader`, which itself has an essentially no-op `close` operation.